PSI_Call.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Create a PSI request (GET)
+ */
+function newPSIRequest(url="http://web.dev.com",device="MOBILE"){
```

Comment:
The default values for the parameters seems wrong here, since if I don't pass a URL I get the results for web.dev. I think this should likely return null or throw.

Reply:
fixed (this was for demo purposes)

Added 
```
/**
 * Showcase how the newPSIRequest function works to be tested from the Apps Script UI
 */
function newPSIRequestDemo(){
  Logger.log(newPSIRequest("http://web.dev.com", "MOBILE"))
}
```